scripts/docker/debian/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM debian:jessie
+
+# Misc Dependencies for build
+RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
+    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
+    apt-get update && apt-get -qqy install\
+	libc6 \
+	libedit2 \
+	libffi6 \
+	libgcc1 \
+	libicu52 \
+	liblldb-3.6 \
```

Comment:
I assume you are talking about `liblldb-3.6`?

This library is used when you want to debug the runtime using LLVM and use SOS (which is a helper plugin that understands how to do runtime specific things like dump method tables, converted the addresses of jit'ed methods back to what method they came from, get information about the GCHeap, etc).

It's not used at runtime by CoreCLR.  If this package doesn't exist, CoreCLR will still work.